File: src/main/java/zaftnotameni/creatania/ponder/CreataniaPonderIndex.java
package zaftnotameni.creatania.ponder;

public class CreataniaPonderIndex {

  public static void register() {
    registerScenes();
    associateTags();
    registerTags();
  }

  public static void associateTags() {
    CreataniaTagMap.associate();
  }
  public static void registerTags() {
    CreataniaPonderTag.register();
  }

  public static void registerScenes() {
    CreataniaPonderScene.register();
  }

}

File: src/main/java/zaftnotameni/creatania/ponder/CreataniaPonderIndex.kt
package zaftnotameni.creatania.ponder

import zaftnotameni.creatania.ponder.CreataniaTagMap.associate

object CreataniaPonderIndex {
  @JvmStatic
  fun register() {
    registerScenes()
    associateTags()
    registerTags()
  }

  fun associateTags() {
    associate()
  }

  fun registerTags() {
    CreataniaPonderTag.register()
  }

  fun registerScenes() {
    CreataniaPonderScene.register()
  }
}